lms/static/js/spec/groups/views/cohorts_spec.js

```diff
@@ -255,6 +277,54 @@ define(['backbone', 'jquery', 'js/common_helpers/ajax_helpers', 'js/common_helpe
                 });
             });
 
+            describe("Course Cohort Settings", function () {
+                it('enable/disable working correctly', function () {
+                    createCohortsView(this, {cohortSettings: createCohortSettings(false)});
+
+                    expect(cohortsView.$('.cohorts-state').prop('checked')).toBeFalsy();
+
+                    cohortsView.$('.cohorts-state').prop('checked', true).change();
+                    AjaxHelpers.expectJsonRequest(
+                        requests, 'PUT', '/mock_service/cohorts/settings',
+                        createMockCohortSettings(true, [], true)
+                    );
+                    AjaxHelpers.respondWithJson(
+                        requests,
+                        createMockCohortSettings(true)
+                    );
+                    expect(cohortsView.$('.cohorts-state').prop('checked')).toBeTruthy();
+
+                    cohortsView.$('.cohorts-state').prop('checked', false).change();
+                    AjaxHelpers.expectJsonRequest(
+                        requests, 'PUT', '/mock_service/cohorts/settings',
+                        createMockCohortSettings(false, [], true)
+                    );
+                    AjaxHelpers.respondWithJson(
+                        requests,
+                        createMockCohortSettings(false)
+                    );
+                    expect(cohortsView.$('.cohorts-state').prop('checked')).toBeFalsy();
+                });
+
+
+                it('Course Cohort Settings Notification View renders correctly', function () {
```

Comment:
I'd go with something like:

    it('shows an appropriate cohort status message', ...